packages/pds/tests/file-uploads.test.ts

```diff
@@ -5,8 +5,6 @@ import { CloseFn, runTestServer, TestServerInfo } from './_util'
 import { Database, ServerConfig } from '../src'
 import DiskBlobStore from '../src/storage/disk-blobstore'
 import * as uint8arrays from 'uint8arrays'
-import * as image from '../src/image'
```

Comment:
I wonder if some big parts of `src/image` can also be nixed at this point (e.g. at least `src/image/server.ts`?)

Reply:
ripped out all the image server + img uri stuff 👌